src/praisonai-agents/praisonaiagents/agent/chat_mixin.py

```
@@ -135,6 +135,50 @@ def _build_system_prompt(self, tools=None):
+                        target_descriptions.append(desc)
+                    
+                    if target_descriptions:
+                        context_parts.append(
+                            f"Reachable delivery targets: {', '.join(target_descriptions)}."
+                        )
+                
+                if context_parts:
+                    system_prompt += f"\n\n## Session Context\n" + "\n".join(context_parts)
+        except (ImportError, Exception):
+            # Session context not available or error, continue without it
+            pass
```

Ruff (0.15.17)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)